src/stories/containers/Finances/SeccionPages/CardsNavigation/CardsNavigation.tsx

```diff
@@ -0,0 +1,38 @@
+import styled from '@emotion/styled';
```

Comment:
the `sectionPages` folder should live inside the `componets` folder of the componets-scoped folder of the page